packages/cadl-ranch-specs/http/payload/multipart/main.tsp

```diff
@@ -9,6 +9,7 @@ namespace Payload.MultiPart;
 
 model MultiPartRequest {
   id: string;
+  temperature: float64;
```

Comment:
Do we have a real case here? Do we need to consider other primitive types e.g null, boolean?

Reply:
Yeah, openai: https://github.com/bterlson/openai-in-typespec/blob/aa3891ccb29a1c8377b6ef3e8f5b5f4825630979/audio/operations.tsp#L19